packages/trpc/server/routers/viewer/organizations/create.handler.ts

```diff
@@ -157,23 +157,25 @@ export const createHandler = async ({ input, ctx }: CreateOptions) => {
 
     const translation = await getTranslation(input.language ?? "en", "common");
 
-    await sendEmailVerification({
-      email: orgOwnerEmail,
-      language: ctx.user.locale,
-      username: ownerProfile.username || "",
-    });
-
-    await sendOrganizationCreationEmail({
-      language: translation,
-      from: ctx.user.name ?? `${organization.name}'s admin`,
-      to: orgOwnerEmail,
-      ownerNewUsername: ownerProfile.username,
-      ownerOldUsername: null,
-      orgDomain: getOrgFullOrigin(slug, { protocol: false }),
-      orgName: organization.name,
-      prevLink: null,
-      newLink: `${getOrgFullOrigin(slug, { protocol: true })}/${ownerProfile.username}`,
-    });
+    !isPlatform &&
+      (await sendEmailVerification({
+        email: orgOwnerEmail,
+        language: ctx.user.locale,
+        username: ownerProfile.username || "",
+      }));
+
```

Comment:
NIT: we should probably move this check inside sendEmailVerification.


  if (isPlatform) {
    log.warn("Skipping Email verification");
    return { ok: true, skipped: true };
  }

Reply:
You're right, but what we're trying to do here is just a temporary fix, if you look at #14911 there we refactor the whole of the create org handler into smaller functions which do just one particular job like the sendEmailVerification you mentioned about.

Reply:
Might have to go with because e2e fails for this

Reply:
@Udit-takkar do we have tests for if email verification is skipped?